src/Keyboard/index.tsx

```diff
@@ -0,0 +1,246 @@
+import Keyboard from 'react-simple-keyboard';
+import 'react-simple-keyboard/build/css/index.css';
+import './Keyboard.css';
+import { useEffect, useLayoutEffect, useRef, useState } from 'react';
+import { createPortal } from 'react-dom';
+import {DISPLAY_WIDTH, DISPLAY_HEIGHT } from '../const/display';
```

Comment:
Just a missing space before `DISPLAY_WIDTH.`

Reply:
Fixed in 52a09ef